src/tutorial/helmholtz.jl

```diff
@@ -45,31 +45,14 @@ println("Running Helmholtz example...") #hide
 # # Uncommented code
 # The code below solves the described Helmholtz eigen problem. The code with detailed comments
 # is provided in the next section.
-using Bcube
-using LinearAlgebra
-
-mesh = rectangle_mesh(21, 21)
-
-degree = 1
-
-U = TrialFESpace(FunctionSpace(:Lagrange, degree), mesh)
-V = TestFESpace(U)
-
-dΩ = Measure(CellDomain(mesh), 2 * degree + 1)
 
-a(u, v) = ∫(∇(u) ⋅ ∇(v))dΩ
-b(u, v) = ∫(u ⋅ v)dΩ
-
-A = assemble_bilinear(a, U, V)
-B = assemble_bilinear(b, U, V)
-
-vp, vecp = eigen(Matrix(A), Matrix(B))
-@show sqrt.(abs.(vp[3:8]))
+# @__PLAIN_CODE__
 
 # # Commented code
 # Load the necessary packages
 using Bcube
 using LinearAlgebra
+using Test #src
```

Comment:
maybe we can move this `using Test` inside the `if get(ENV...`?

Reply:
I already tried and it does not work... I don't know why